docs/en/stack/monitoring/esms.asciidoc

```diff
@@ -0,0 +1,208 @@
+[role="xpack"]
+[[esms]]
+== {esms}
+
+The {esms} ({esms-init}) is a monitoring cluster on {ecloud}. Elastic provides
+and maintains {esms-init} for self-managed commercial customers. If you send
+your monitoring data to {esms-init}, it can also be used by Elastic support to
+provide better and faster incident resolution.
+
+NOTE: You must obtain your {esms-init} cluster URLs and credentials from the
+Elastic support team.
+
+[discrete]
+[[esms-elasticsearch]]
+=== Collecting monitoring data about {es}
+
+There are two methods for collecting and sending data about the health of your
+production cluster to {esms-init}: {metricbeat} or collectors and exporters.
+
+TIP: If you want to monitor {ls}, you must use collectors and exporters to route
+data from the production cluster to {esms-init}. Otherwise, it is simplest to
+use {metricbeat}. 
+
+To use {metricbeat}:
+
+. If your production cluster is stopped,
+{ref}/starting-elasticsearch.html[start it].
+
+. Configure the following settings on each node in the production cluster:
```

Comment:
For this step, could we describe to the user _why_ they should configure these settings, kinda like we've done in https://www.elastic.co/guide/en/elasticsearch/reference/master/configuring-metricbeat.html:

> Enable the collection of monitoring data. Set `xpack.monitoring.collection.enabled` to `true` on each node in the production cluster. By default, it is is disabled (`false`).
> Disable the default collection of Elasticsearch monitoring metrics. Set `xpack.monitoring.elasticsearch.collection.enabled` to `false` on each node in the production cluster.

Reply:
I had combined those two settings steps for the sake of brevity, but it's easy enough to just re-use them as-is from the Elasticsearch steps.  I've made that change.